site-manage/site-manage-participant-helper/src/java/org/sakaiproject/site/tool/helper/participant/impl/SiteAddParticipantHandler.java

```diff
@@ -336,6 +337,10 @@ public String getServerConfigurationString(String param, String defaultValue)
     public String processCancel() {
         ToolSession session = sessionManager.getCurrentToolSession();
         session.setAttribute(ATTR_TOP_REFRESH, Boolean.TRUE);
+
+         // Go to Site Info landing page on 'Cancel'
+        session.setAttribute(SiteConstants.STATE_TEMPLATE_INDEX, SiteConstants.SITE_INFO_TEMPLATE_INDEX);
```

Comment:
This seems to be repeated a few times. Maybe a utility method like goToPage(session, SiteConstants.SITE_INFO_TEMPLATE_INDEX) would be useful?